internal/worker/httpserverargs/worker.go

```diff
@@ -85,9 +85,16 @@ func newWorker(ctx context.Context, cfg workerConfig) (worker.Worker, error) {
 		return nil, errors.Trace(err)
 	}
 
+	st, err := w.cfg.statePool.SystemState()
```

Comment:
We've really only relocated this block in this patch. No dropping of calls...

Reply:
That's true. However, it is only called once when the worker starts, instead of each time the apiserver receives a request.

But I guess what I was getting at is it's a step to removing state as a dependency from `apiserver/stateauthenticator`